index.js

```diff
@@ -366,9 +366,13 @@ NYC.prototype.wrap = function (bin) {
 }
 
 NYC.prototype.generateUniqueID = function () {
-  return md5hex(
-    process.hrtime().concat(process.pid).map(String)
-  )
+  const hash = crypto.createHash('sha256')
```

Comment:
This is valid and I don't have an issue with it, just have to ask what about using the `uuid` module instead?  That might actually be more correct for this purpose but I'm not sure it's too important.

Reply:
@coreyfarrell @addaleax great point, in this context it seems like using `uuid.v4()` would be reasonable? this doesn't solve the issue with caching-transform performance, which I think is worth discussing further in that thread.